todo_app/tests/test_e2e.py

```diff
@@ -2,20 +2,24 @@
 import pytest
 from threading import Thread
 from time import sleep
-import requests
 from selenium import webdriver
 from selenium.webdriver.firefox.options import Options
 from dotenv import load_dotenv
 from todo_app import app
+from todo_app.todo import Item
+import pymongo
 
 @pytest.fixture(scope='module')
-def app_with_temp_board():
+def app_with_temp_db():
     # Load our real environment variables
     load_dotenv(override=True)
 
     # Create the new board & update the board id environment variable
-    board_id = create_trello_board()
-    os.environ['TRELLO_BOARD_ID'] = board_id
+    test_db = items()
```

Comment:
I'm not sure I follow what this step is about; it just makes a query to an empty DB? We could just let the DB be created on the fly during the test instead

Reply:
I've removed this line